devops/scripts/generate_test_matrix.js

```diff
@@ -0,0 +1,36 @@
+module.exports = ({ core, process }) => {
+  const fs = require('fs');
+  fs.readFile('./test_configs.json', 'utf8', (err, data) => {
+    if (err) {
+      console.log(`Error reading file from disk: ${err}`);
+    } else {
+      const testConfigs = JSON.parse(data);
+      const inputs = JSON.parse(process.env.GHA_INPUTS);
+
+      const ltsConfigs = inputs.lts_config.split(';');
+
+      console.log(inputs);
+      console.log(ltsConfigs);
+
+      const enabledLTSConfigs = [];
+
+      testConfigs.lts.forEach(v => {
+        console.log(v);
+        if (ltsConfigs.includes(v.config)) {
+          console.log("BOO");
```

Comment:
What is "BOO"? Could add a comment for this line explaining the choice for this message?

Reply:
It's just a lack of proper debugger for these scripts. Removed debug output.